NangPaGo-be/src/main/java/com/mars/NangPaGo/domain/community/service/CommunityLikeService.java

```diff
@@ -23,16 +23,21 @@ public class CommunityLikeService {
     private final CommunityRepository communityRepository;
     private final UserRepository userRepository;
 
-    @Transactional(readOnly = true)
     public boolean isLiked(Long id, String email) {
         return communityLikeRepository.findByEmailAndCommunityId(email, id).isPresent();
     }
 
+    public long countLikes(Long id) {
```

Comment:
메서드명에 `get`이 들어가주면 좋을 것 같아요.

```suggestion
    public long getLikeCount(Long id) {
```